src/cli/cli_dataset.cpp

```diff
@@ -51,12 +51,12 @@ otError Dataset::Print(otOperationalDataset &aDataset)
 {
     if (aDataset.mComponents.mIsPendingTimestampPresent)
     {
-        OutputLine("Pending Timestamp: %lu", aDataset.mPendingTimestamp);
+        OutputLine("Pending Timestamp: %lu", aDataset.mPendingTimestamp.mSeconds);
```

Comment:
we may also want to print ticks and `mAuthoritative` bit ?

```suggestion
        OutputLine("Pending Timestamp: %lu.%lu.%c", aDataset.mPendingTimestamp.mSeconds, aDataset.mPendingTimestamp.mTicks, 
        aDataset.mPendingTimestamp.mAuthoritative ? 'A' : 'a');
```

Reply:
I would agree I prefer printing the full timestamp. But I don't want to break any existing tests or public openthread users as I think there are someone (e.g. Nordic) depending on the CLI format.

I will keep the CLI interface unchanged and leave it to a separate PR.